CHANGELOG.md

```
@@ -183,7 +194,8 @@
 
 - Initial release
 
-[unreleased]: https://github.com/hkrn/ndmf-vrm-exporter/compare/1.2.5...HEAD
+[unreleased]: https://github.com/hkrn/ndmf-vrm-exporter/compare/1.2.6...HEAD
+[1.2.5]: https://github.com/hkrn/ndmf-vrm-exporter/compare/1.2.5...1.2.6
 [1.2.5]: https://github.com/hkrn/ndmf-vrm-exporter/compare/1.2.4...1.2.5
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Duplicate link or image reference definition: "1.2.5"

(MD053, link-image-reference-definitions)